src/coreclr/src/pal/tests/palsuite/c_runtime/_fdopen/test1/test1.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include <palsuite.h>
 
-const char* cTestString = "one fish, two fish, read fish, blue fish.";
+#define cTestString "one fish, two fish, read fish, blue fish."
```

Comment:
Why are these code changes necessary? e.g. ~9 days ago (https://github.com/dotnet/runtime/pull/41622), master branch was building with `./src/coreclr/build-runtime.sh -skipgenerateversion -nopgooptimize -cmakeargs -DCLR_CMAKE_BUILD_TESTS=1` on Ubuntu, macOS and OpenIndiana hosts.

Reply:
With this change nearly everything is building into 1 large binary. These symbols were interfering with linking, and while they could be fixed by making them static, I've always found statics to be annoying to debug, so I didn't use static.